packages/components/assets/save.svg

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?><!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
-<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" fill="none">
+<!-- <?xml version="1.0" encoding="utf-8"?>Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
+<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" style="filter: invert(100%) sepia(100%) saturate(0%) hue-rotate(174deg) brightness(102%) contrast(102%);">
```

Comment:
Why is a filter being applied here? Applying these in the svg file causes them to change everywhere

Reply:
Replaced these filters on the component files themselves with inline fills where they are used